package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "chromeback",
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Comment:
we should publish at 0.1.0 so leave this at 0.0.0 so `npm version minor` bumps to 0.1.0

reason being - no tests